src/lottie-player.ts

```diff
@@ -165,6 +191,25 @@ const _downloadFile = (fileName: string, blob: Blob) => {
   document.body.removeChild(link);
 }
 
+let _initStatus = InitStatus.IDLE;
+const _initModule = async (engine: Renderer) => {
+  if (engine === Renderer.SW) {
```

Comment:
Recommend to add a comment something like this:
//thorvg doesn't do anything in the module init(). Skip ASAP.

Reply:
@hermet 
We can remove the line, but `software renderer doesn't do anything in the module init()` is not true.

Without engine condition, it will run WG initialization even if only SW module requested. `init()` binding function is out of TvgLottieAnimation class, we don't know when to skip.

Reply:
@tinyjin My point is to add a comment there for the exceptional handling. we do return ASAP, since we know sw renderer doesn't do anything in the module init. (only wg does something)